bbb-graphql-server/bbb_schema.sql

```diff
@@ -2186,6 +2186,11 @@ SELECT *
 FROM "caption"
 WHERE "createdAt" > current_timestamp - INTERVAL '5 seconds';
 
+--------See all captions from a meeting
+CREATE OR REPLACE VIEW "v_caption_persistent" AS
```

Comment:
I suggest to name it `v_caption_history` as `_history` is the pattern we have been using for this situation.